src/main/resources/modules/covid19.json

```diff
@@ -10,35 +10,32 @@
   "states": {
     "Initial": {
       "type": "Initial",
-      "direct_transition": "Wait Until Outbreak"
+      "direct_transition": "Wait Until Exposure"
     },
     "Terminal": {
       "type": "Terminal"
     },
-    "Wait Until Outbreak": {
-      "allow": {
-        "condition_type": "Date",
-        "operator": "==",
-        "year": 2020
-      },
-      "type": "Guard",
-      "direct_transition": "Wait Until Exposure"
-    },
     "Wait Until Exposure": {
-      "range": {
-        "low": 1,
-        "high": 90,
+      "type": "Delay",
+      "exact": {
+        "quantity": 4,
         "unit": "days"
       },
-      "type": "Delay",
-      "distributed_transition": [
+      "lookup_table_transition": [
+        {
+          "transition": "Wait Until Exposure",
+          "default_probability": "1",
+          "lookup_table_name": "covid19_prob.csv"
+        },
         {
-          "distribution": 0.2,
-          "transition": "Call Infection Submodule"
+          "transition": "Call Infection Submodule",
+          "default_probability": "0",
```

Comment:
I just want to point out that you have a `default_probability` of zero in both transitions, which means if the patient does not match any row in your `covid19_prob.csv` file, then they won't go to either `Call Infection Submodule` or `Terminal`... you should probably have them default somewhere.

Even if the `covid19_prob.csv` file _currently_ has an exhaustive list of matches, I think it is a "best practice" to have defaults.

Reply:
It does have a default transition, back to Wait Until Exposure.